src/webgpu/api/operation/resource_init/buffer.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+
+export const description = `
+Test uninitialized buffers are initialized to zero when read.
```

Comment:
or written, for example with atomics (technically a combined read / write but I think it's good to call it out).